.rubocop.yml

```diff
@@ -5,9 +5,13 @@ inherit_gem:
 
 AllCops:
   NewCops: enable
+  SuggestExtensions: false
 
 Rails/RefuteMethods:
   Enabled: false
 
 Style/Documentation:
   Enabled: false
+
+Rails/RakeEnvironment:
+  Enabled: false
```

Comment:
I was getting a rubocop error [for this line](https://github.com/github/erblint-github/pull/8/files#diff-c214978886585f469c549fe44e7c4681eaae0e8be954e7ce91aef993bda0ddc3R4), [Rails/RakeEnvironment](https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsrakeenvironment), but disabling it because this repo isn't running a Rails app.